spec/features/accessibility/visitor_pages_spec.rb

```diff
@@ -40,5 +44,6 @@
     expect(page).to be_axe_clean.according_to :section508, :"best-practice", :wcag21aa
     expect(page).to label_required_fields
     expect(page).to be_uniquely_titled
+    expect(page).to have_valid_markup
```

Comment:
same comment as #7642, maybe we make a singular helper?

Reply:
> same comment as #7642, maybe we make a singular helper?

Updated in rebased ea59a28b4.